lib/insert/insert_supporter.rb

```diff
@@ -14,8 +14,9 @@ def self.create_or_update(np_id, data, update=false)
     })
     address_keys = ['name', 'address', 'city', 'country', 'state_code']
     custom_fields = data['customFields']
+    tags = data['tags']
```

Comment:
Do we have any tests for InsertSupporter? If so, it would be good to test this.

Reply:
@wwahammy we don't :( But I think it's a good opportunity to start testing this module, what do you think?

Reply:
Hmmm. Ya, a limited set of tests would be good. Over time, I want to totally scrap that code but it'll be with us a for a while.

Reply:
I just added a few tests, and I did a smoke test again with the most recent changes.

Reply:
@wwahammy I was able to update that line now :)